src/App.tsx

```diff
@@ -15,48 +16,133 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+  let resultList;
+
+  switch (sortType) {
+    case SortType.ALPABET:
+      resultList = visibleGoods.sort((goodA, goodB) => (
+        goodA.localeCompare(goodB)
+      ));
+      break;
+
+    case SortType.LENGTH:
+      resultList = visibleGoods.sort((goodA, goodB) => (
+        goodA.length - goodB.length
+      ));
+      break;
+    default:
+      resultList = visibleGoods;
+      break;
+  }
```

Comment:
use `switch` condition inside `sort` method to avoid repeating this method

Reply:
agree, but it's not critical